ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -8,3 +8,34 @@
 @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
 def layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=None):
     return ivy.layer_norm(x, normalized_shape, weight, bias, epsilon)
+
+
+# batch_norm
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def batch_norm(
```

Comment:
Added this batch_norm function.

Reply:
As mentioned earlier, this can be removed as you already have a separate PR for this